server/settings.py

```diff
@@ -152,6 +152,10 @@ def env(variable, fallback_value=None):
         "task": "liveblog.blogs.tasks.remove_deleted_blogs",
         "schedule": crontab(hour="*/1"),  # run every hour
     },
+    "bandwidth:update": {
+        "task": "liveblog.bandwidth.tasks.fetch_bandwidth_usage",
+        "schedule": crontab(minute=0, hour="*/1"),  # every hours
+    },
```

Comment:
Similarly here, this task should only be scheduled if the subscription level requires it. Since we cannot access the instance settings at this point (before scheduling it), we either add a condition inside the task itself that would check the setting or we schedule it only if the subscription level is "liveblog-go".